src/workspacer.Shared/Workspace/IWorkspace.cs

```diff
@@ -1,147 +1,127 @@
 
 namespace workspacer
 {
     /// <summary>
     /// IWorkspace provides a common interface for workspace-related operations.
-    /// workspaces logically contain a set of windows, and allow callers to interact with the 
+    /// Workspaces logically contain a set of windows, and allow callers to interact with the 
     /// windows via a set of methods, and all the organization of the windows via layout engines
     /// </summary>
     public interface IWorkspace
     {
         /// <summary>
-        /// name of the workspace
+        /// Name of the workspace
         /// </summary>
         string Name { get; set; }
 
         /// <summary>
-        /// name of the currently active layout
+        /// Name of the currently active layout
         /// </summary>
         string LayoutName { get; }
 
         /// <summary>
-        /// set of windows that are contained within the workspace
+        /// Set of windows that are contained within the workspace
         /// </summary>
         IEnumerable<IWindow> Windows { get; }
 
         /// <summary>
-        /// set of windows that are contained within the workspace and which workspacer can layout.
+        /// Set of windows that are contained within the workspace and which workspacer can layout.
         /// </summary>
         IList<IWindow> ManagedWindows { get; }
 
         /// <summary>
-        /// currently focused window in the workspace, or null if there is no focused window in the workspace
+        /// Currently focused window in the workspace, or null if there is no focused window in the workspace
         /// </summary>
         IWindow FocusedWindow { get; }
 
         /// <summary>
-        /// the last focused window in the workspace
+        /// The last focused window in the workspace
         /// </summary>
         IWindow LastFocusedWindow { get; }
 
         /// <summary>
-        /// whether the workspace is currently indicating (flashing)
+        /// Whether the workspace is currently indicating (flashing)
         /// </summary>
         bool IsIndicating { get; set; }
 
-        void AddWindow(IWindow window, bool layout = true);
-        void RemoveWindow(IWindow window, bool layout = true);
-        void UpdateWindow(IWindow window, WindowUpdateType type, bool layout = true);
-
-        /// <summary>
-        /// close the currently focused window
-        /// </summary>
-        void CloseFocusedWindow(); // mod-shift-c
-
-        /// <summary>
-        /// rotate to the previous layout
-        /// </summary>
-        void PreviousLayoutEngine(); // mod-space
-
-        /// <summary>
-        /// rotate to the next layout
-        /// </summary>
-        void NextLayoutEngine(); // mod-space
-
-        /// <summary>
-        ///  reset the active layout
-        /// </summary>
-        void ResetLayout(); // mod-n
-
-        /// <summary>
-        /// focus the last focused window
-        /// </summary>
-        void FocusLastFocusedWindow(); 
```

Comment:
Were these methods actually removed, or just alphabetically re-arranged (with code-docs added for some)?

If no actual change was done here, having these changes in among the rest makes reviewing the actual changes harder.